from simcore_ai.exceptions.base import SimCoreError


class IdentityError(SimCoreError): ...